### ISSUE TITLE
Too much storage required

### ISSUE THREAD

**Issue by AlvaroCavalcante** [ferreirafabio/video2tfrecord#39]
Hello everyone, the video2tfrecord library is working very well, great work!  

I've done just some small modifications to run the code on TF 2, and this is one of my suggestions, we could detect the TensorFlow version and change the code dynamically. 

 Another interesting point is the compression of the video frames. Using the default code, each TFRecord with 5 videos and 25 frames got about 340MB. With this proportion, just my test set would get about 21GB...

To solve this, I changed the code to generate the byte image from this: ```image.tostring()``` to this: ```tf.image.encode_jpeg(image).numpy()```. With this change, the same TFRecord just got about 18MB which is much more efficient to store video datasets. This could be a second modification...

What do you think about this?

**Comment by ferreirafabio** [comment]
Hi Alvaro,

my guess would be that by using "encode_jpeg", you're compressing the image before storing it. While this makes a lot of sense, I feel that this should by a hyperparameter "compress_images" that is set to True or False by default. If set to False, it should use the image.tostring() method. Feel free to create a PR and I'll check it :-) thanks a lot again for your efforts!

**Comment by AlvaroCavalcante** [comment]
Great @ferreirafabio, I've done the PR!
